gcp_variant_transforms/libs/sample_info_table_schema_generator.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2019 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Generates sample info table schema."""
+
+from apache_beam.io.gcp.internal.clients import bigquery
+
+from gcp_variant_transforms.libs import bigquery_util
+
+SAMPLE_ID = 'sample_id'
+FILE_PATH = 'file_path'
+
+
+def generate_schema():
+  # type: () -> bigquery.TableSchema
+  schema = bigquery.TableSchema()
+  schema.fields.append(create_sample_id_field())
+  schema.fields.append(bigquery.TableFieldSchema(
```

Comment:
Seems weird that first field is added via create_sample_id_field() method, while the second one is added directly. I think both should be added in the same way (whichever you find more appropriate - your discretion).